src/helpers/getWholePageList.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from "react";
+import {routeProps} from "../data/routeProps";
+
+export const getWholePageList = (): JSX.Element [] => {
```

Comment:
This component is redundant. Use map in component where you need it (not in tests)